app.R

```diff
@@ -0,0 +1,1267 @@
+      ),
+      tags$div(
+        class = "sidebar-logo",
+        boastUtils::sidebarFooter()
+      )
+    ),
+    ### Create the content ----
+    dashboardBody(
+      tabItems(
+        #### Set up the Overview Page ----
+        tabItem(
+          tabName = "overview",
+          withMathJax(),
+          h1("Chi-Square Goodness-fit-Test"),
+          # Please reformat long lines of code; the line below is an example
+          p("In this app you will explore the Chi-Square Goodness-fit-Test with
+            simulations. The test is applied when you have categorical variables
+            from a population."),
+          br(),
+          h2("Instructions"),
+          tags$ul(
+            tags$li("Select one of the scenarios for the proportion in each category (Equal Probabilities or Different Probabilities)."),
+            tags$li("Move the sliders to change the values of number of observations, number of categories, and number of simulations."),
+            tags$li("A p-value is calculated and plotted for each simulation. You can click a p-value on the plot to see the summary table for that dataset."),
+            tags$li("When there are more than 50 simulations, only a histogram of p-values is shown."),
+            tags$li("You can use the Explore page to explore your own data and null hypothesis.")
+          ),
+          br(),
+          ##### Go Button ----
+          div(
+            style = "text-align: center",
+            bsButton(inputId = "overviewBotton",
+                     label = "Prerequisites",
+                     size = "large",
+                     icon = icon("bolt"),
+                     style = "default"
+            )
+          ),
+          ##### Set Acknowledgements Part ----
+          br(),
+          br(),
+          h2("Acknowledgements"),
+          p("This app was developed and coded by Jinglin Feng. Later modified by Anna (Yinqi) Zhang.
+            Special thanks to Alex Chen, Yuxin Zhang and Dr. Neil Hatfield for help on some programming issues.
+            The most recent version was updated by Yudan Zhang.",
+            div(class = "updated", "Last Update: 5/26/2021 by YDZ.")
+          )
+        ),
+        #### Set up the Prerequisites Page ----
+        tabItem(
+          tabName = "prerequisites",
+          withMathJax(),
+          h1("Prerequisites"),
+          p("When an analyst attempts to fit a statistical model to observed data,
+         they may wonder how close are the observed values to those which would be expected.
+         One statistical test that addresses this issue for categorical data is the chi-square goodness-of-fit test."),
+         p("If the observed and expected values are not close, 
+           then the computed test statistic is large and the model is a poor fit to the data."),
+  
+         box(
+           title = strong("Hypotheses"), # Remove all instances of strong in titles
```

Comment:
You left in the comment to remove all strongs, but haven't yet. Is this something you changed your mind on?